lib/nfs_common.pm

```diff
@@ -277,9 +277,19 @@ sub check_y2_nfs_func {
     stop_service();
     # we need to cleanup the nfs settings after service check was done.
     if ($stage eq 'after') {
-        zypper_call 'rm yast2-nfs-server nfs-kernel-server', timeout => 480;
-        script_run ':> /etc/exports';
+        zypper_call 'rm yast2-nfs-server', timeout => 480;
```

Comment:
Do we still need timeout =480 here?

Reply:
leave it as it was. to make sure it will be 100% finished

Reply:
cool, thanks